react/AutocompleteBlock.tsx

```diff
@@ -115,9 +116,17 @@ const AutocompleteBlock: StorefrontFunctionComponent<any &
 
   const { selectedItem, quantitySelected, unitMultiplier } = state
   const callAddToCart = async (items: any) => {
+      let currentItemsInCart = orderForm.orderForm.items
     const mutationResult = await addToCart({
       variables: {
         items: items.map((item: any) => {
+            let existsInCurrentOrder = currentItemsInCart.filter(
+              el => el.id === item.id.toString()
+            )
+            if (existsInCurrentOrder.length > 0) {
```

Comment:
`if (existsInCurrentOrder?.length > 0) {`

Reply:
In the condition where we check if the item exists in the current order it was not possible to use "existsInCurrentOrder?.length > 0" because the folowing error was happening "property 'lenght' does not exist on type 'Item'."

So I changed the condition to just "if (existsInCurrentOrder) {" considering that the "existsInCurrentOrder" is the actual item.